statsmodels/robust/tests/test_scale.py

```diff
@@ -56,6 +56,10 @@ def test_mad(self):
         m = scale.mad(self.X)
         assert_equal(m.shape, (10,))
 
+    def test_mad_empty(self):
+        empty = np.asarray([])
+        assert_equal(np.isnan(scale.mad(empty),True)
```

Comment:
you can just use `assert_(np.isnan(scale.mad(empty))`
with assert_ from numpy.testing

Reply:
just seeing this now, I'll add that in if the build passes this time. I had missed a `)` inside to close the `np.isnan` hopefully passes all tests now